### PR TITLE
Change token permissions to match update in core v14-beta

### PR DESCRIPTION
**Pull request by MuncleUscles**

### CODE REVIEW

File: src/modules/account/RadixTokenDefinitionAccountSystem.ts [hunk 1]
```diff
@@ -98,12 +98,10 @@ export class RadixTokenDefinitionAccountSystem implements RadixAccountSystem {
         tokenDefinition.granularity = particle.granularity
 
         const mintPermissions = particle.permissions.mint
-        if (mintPermissions === RadixTokenPermissionsValues.SAME_ATOM_ONLY || mintPermissions === RadixTokenPermissionsValues.GENESIS_ONLY) {
+        if (mintPermissions === RadixTokenPermissionsValues.TOKEN_CREATION_ONLY || mintPermissions === RadixTokenPermissionsValues.NONE) {
             tokenDefinition.tokenSupplyType = RadixTokenSupplyType.FIXED
-        } else if (mintPermissions === RadixTokenPermissionsValues.TOKEN_OWNER_ONLY) {
+        } else if (mintPermissions === RadixTokenPermissionsValues.TOKEN_OWNER_ONLY || mintPermissions === RadixTokenPermissionsValues.ALL) {
```
Comment (laszlourszuly) [inline src/modules/account/RadixTokenDefinitionAccountSystem.ts]: What does `RadixTokenPermissionsValues.ALL` mean? Is it like a super user (who has all permissions)?

**Review by laszlourszuly — APPROVED** [review]
Looks good to me. I haven't looked at the corresponding changes in core, though, but I'm confident you have :)

File: src/modules/account/RadixTokenDefinitionAccountSystem.ts (hunk 1 above)
Comment (MuncleUscles) [reply src/modules/account/RadixTokenDefinitionAccountSystem.ts]: It means anyone can mint it, like the POW token currently(though I think we're making it into something else)

Comment (laszlourszuly) [reply src/modules/account/RadixTokenDefinitionAccountSystem.ts]: Ok, but is that what we want to do? Allow anyone, not only the creator, to mint new tokens? I suppose it's an interim solution.

Comment (MuncleUscles) [reply src/modules/account/RadixTokenDefinitionAccountSystem.ts]: If you create a token with that permission, yeah. It's not really exposed through the library, and I'm not sure if there are many use cases for that, but I don't see why not allow such an option.

Comment (MuncleUscles) [reply src/modules/account/RadixTokenDefinitionAccountSystem.ts]: Correction, it is possible to make such a token through the js lib, but it's not trivial, you have to know what you're doing to do it

**Pull request merged**